localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -1026,14 +1026,14 @@ geometry_msgs::msg::PoseWithCovarianceStamped NDTScanMatcher::align_pose(
   const double stddev_pitch = std::sqrt(covariance(4, 4));
 
   // Since only yaw is uniformly sampled, we define the mean and standard deviation for the others.
-  const std::vector<double> sample_mean{
+  std::vector<double> sample_mean{
     initial_pose_with_cov.pose.pose.position.x,  // trans_x
     initial_pose_with_cov.pose.pose.position.y,  // trans_y
     initial_pose_with_cov.pose.pose.position.z,  // trans_z
     base_rpy.x,                                  // angle_x
     base_rpy.y                                   // angle_y
   };
-  const std::vector<double> sample_stddev{stddev_x, stddev_y, stddev_z, stddev_roll, stddev_pitch};
+  std::vector<double> sample_stddev{stddev_x, stddev_y, stddev_z, stddev_roll, stddev_pitch};
```

Comment:
Can we restore const in these variables?

Reply:
Also https://github.com/autowarefoundation/autoware.universe/pull/7364/files#diff-2f9b5f8852197134069202820aee0565d91b6c85c2dc2e5725f3b8f8025cc10fR1029

Reply:
sorry for the careless commits, I fixed it at 0decbc2 .